src/site/content/en/blog/maskable-icon/index.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Better PWA icons with maskable icons
+subhead: A new icon format to use adaptive icons on supporting platforms.
+description: |
+  A description of the article that will appear in search results.
+authors:
+  - tigeroakes
+date: 2019-12-13
+hero: hero.png
+alt: Icons contained inside white circles compared to icons covering its entire circle
+tags:
+  - post
+  - capabilities
+  - progressive-web-apps
+---
+
+## What are maskable icons? {: #what }
+
+If you've installed a Progressive Web App on a recent Android phone, you might notice the icon shows up with a white background. Android Oreo introduced adaptive icons, which display app icons in a variety of shapes across different device models. Icons that don't follow this new format are given white backgrounds.
+
+<figure class="w-figure">
```

Comment:
You may also want to add the `w-figure--center` class to center these images. That seems to look a bit nicer.